webpack.config.js

```diff
@@ -3,7 +3,8 @@ var path = require("path");
 
 var entries = {
   background: "./webextension/background.js",
-  manage: "./webextension/manage.js"
+  manage: "./webextension/manage.js",
+  "babel-polyfill"
```

Comment:
Feel free to revert or push over my crummy merge attempt here. Sorry for stepping on toes.